engine/gamesys/src/gamesys/scripts/script_camera.cpp

```diff
@@ -52,15 +52,14 @@ namespace dmGameSystem
     static int Camera_AcquireFocus(lua_State* L)
     {
         DM_LUA_STACK_CHECK(L, 0);
-        dmGameObject::HInstance instance = CheckGoInstance(L);
+        (void)CheckGoInstance(L); // left to check that it's not called from incorrect context.
```

Comment:
Now that we no longer _need_ the instance, the question is: Should we still call the function?
Currently, it helps us checking that we're in a go script, and possibly within a life cycle function.